packages/core/src/utils/Utils.ts

```diff
@@ -822,6 +822,22 @@ export class Utils {
     return !!GroupOperator[key];
   }
 
+  static hasNestedKey(object: Dictionary, key: string): boolean {
```

Comment:
this sounds more accurate i guess?

```suggestion
  static hasNestedKey(object: unknown, key: string): boolean {
```

Reply:
Well hasnestedkey only works on object types. If typescript compiles with unknown, it should be fine.

Reply:
It works with arrays too, you should not represent array via `Dictionary` (I know its assignable, that's not really changing anything).